### PR TITLE
fix: video widget not visible in widget pane when explorer is pinned

### LINKED ISSUE #16119: [Bug]-[800]:Video & TreeSelect widgets not visible in the entity Explorer once sidebar is closed
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Once we click on `Close Sidebar` icon, Video & TreeSelect widgets disappear from the entity explorer.
Note: The widgets appear when we search for them in the search bar but not visible in the list of widgets.

https://www.loom.com/share/9a273f9c5b1544bc9fbbd3ca54bac2c7

### Steps To Reproduce

1. Go to any application.
2. In the Entity explorer click on the Widgets tab.
3. At the bottom you'll see TreeSelect & Video widgets.
4. Now click on the `Close Sidebar` (three lines) icon next to the appsmith icon on the top left corner.
5. Notice that TreeSelect & Video widgets disappear from the widget list

### Public Sample App

_No response_

### Version

Cloud

### PR DESCRIPTION
**Pull request by akash-codemonk**
## Description

When the explorer is set to float it's css position value is set to `position: fixed`.  With this position value it no longer respects the height of the parent(the height of the parent is set to 100vh -  the header height - bottom bar height). So when the height exceeds it overflows below the bottom bar.
To fix this we set the height to `height: inherit` so that it inherits the height of the parent even when it is set to `position: fixed`. 

Release

https://user-images.githubusercontent.com/67054171/194006249-cbcfd909-f718-42a8-b8aa-ee6420cb8804.mov

Post fix

https://user-images.githubusercontent.com/67054171/194006335-76901e52-c82a-4b28-8fc4-a9b2b1ddf1e7.mov

Fixes #16119

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Cypress

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by jsartisan — APPROVED** [review]

File: app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/ExplorerTests/Entity_Explorer_Widgets_spec.js [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
+const ee = ObjectsRegistry.EntityExplorer;
 
 describe("Entity explorer tests related to widgets and validation", function() {
   beforeEach(() => {
     cy.addDsl(dsl);
   });
 
+  it("Widgets visibility in widget pane", function() {
+    ee.NavigateToSwitcher("widgets");
+    cy.get("[data-cy='widget-sidebar-scrollable-wrapper']").scrollTo("bottom");
```
[the remark this answers is not in the document]
Comment (akash-codemonk) [reply app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/ExplorerTests/Entity_Explorer_Widgets_spec.js]: fixed

**Review by appgittest — APPROVED** [review]

**Review by eco-monk — APPROVED** [review]

**Pull request merged**